### PR TITLE
fix: update README docs to standardized templates

### PR DESCRIPTION
**Pull request by NGL321**
## Description of changes:
Updates repo READMEs/Documentation to new AWS SDK repo standards
- Moves learning resources to separate document
- Adds TOC to READMEs
- Adds standardized notice file
- Adds "Getting Help" and "Feedback and Contributing" sections to main README

---

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.*

### CODE REVIEW

File: Docs/Learning_Resources.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
+# Learning Resources
```
Comment (normj) [inline Docs/Learning_Resources.md:1]: Is there a reason this is being put into a separate file? If we really think it is better to be in a separate file then we should have a link or something on the main README for people to notice it.

Comment (NGL321) [reply Docs/Learning_Resources.md:1]: Moving the learning resources to their own file is part of the new standard. I linked it in the main README under `Getting Help`

File: NOTICE [hunk 2]
```diff
@@ -0,0 +1,5 @@
+AWS Lambda for .NET Core 
+Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+This product includes software developed by
+Amazon Technologies, Inc (http://www.amazon.com/).
```
Comment (costleya) [inline NOTICE:5]: Is this official messaging?

File: README.md [hunk 3]
```diff
@@ -239,7 +192,14 @@ To create a function, run the following command
 dotnet new lambda.EmptyFunction --name BlogFunction --profile default --region us-east-2
 ```
 
-
 ### Yeoman (Deprecated)
```
Comment (costleya) [inline README.md:195]: Do we still want to have this section?

**Pull request merged**